### DISCUSSION TITLE
Office hours: 20th April

### DISCUSSION THREAD

**Discussion by amyodea** [twilio-labs/paste#3174] [General] [answered]
The specific questions I have are...

- Best pattern for a wizard flow, how to cancel out of this
- Button Toggle component feedback 
- Error state for a verification code input box 
- Any comments around use of One Twilio design theme? 

Background: Designing the experience for viewing the user profile. 
Stage of design: Exploration
Links to mockups: https://www.figma.com/file/ifEqqrM315oky4TTADledF/Unified-Profile?node-id=95%3A176672&t=cFrc7GMWFoq9J5xj-1

**Comment by TheSisb** [comment, marked as answer]
Hey @amyodea - just wanted to share the office hours recording with you in case you'd like to revisit any of our conversations. [Here's the video](https://www.loom.com/share/963f91def5cb4ba481ec52252b96fb5d), and the password is posted in the #help-design-system channel. Thanks again for coming to office hours!

**Reply by amyodea** [reply]
Thank you, the session was super helpful 🙏
